frontend/src/main/components/BasicCourseSearch/CourseInstructorSearchForm.js

```diff
@@ -0,0 +1,82 @@
+import { useState } from "react";
+import { Form, Button, Container, Row, Col } from "react-bootstrap";
+import { quarterRange } from "main/utils/quarterUtilities";
+import { useSystemInfo } from "main/utils/systemInfo";
+import SingleQuarterDropdown from "../Quarters/SingleQuarterDropdown";
+import { useBackend  } from "main/utils/useBackend";
+
+const CourseInstructorSearchForm = ({ fetchJSON }) => {
+
+  const { data: systemInfo } = useSystemInfo();
+
+  // Stryker disable OptionalChaining
+  const startQtr = systemInfo?.startQtrYYYYQ || "20211";
+  const endQtr = systemInfo?.endQtrYYYYQ || "20214";
+  // Stryker enable OptionalChaining
```

Comment:
```suggestion
  // Stryker restore OptionalChaining
```

Please see: https://ucsb-cs156-s23.slack.com/archives/C050779M19P/p1685726928710669